src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelist.java

```diff
@@ -44,12 +44,15 @@
 import javax.annotation.Nonnull;
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import groovy.lang.GroovySystem;
 import org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException;
 
 /**
  * Whitelist based on a static file.
  */
 public final class StaticWhitelist extends EnumeratingWhitelist {
+    private static final String G2_PREFIX = "g2";
+    private static final boolean GROOVY2 = GroovySystem.getVersion().startsWith("2.");
```

Comment:
:ant: I think you want to use `VersionNumber` or similar to apply the same behavior (unless and until overridden) in Groovy 3.x.